### ISSUE TITLE
Embedd tasking manger functionality inside microsite

### ISSUE THREAD

**Issue by thomas-maschler** [crowdcover/logging-roads#5]
All tasks should be accessible from the micro side. 
It should be possible to lock/ unlock tasks from within micro side.
It should be possible to display tasks from several projects at once.

**Comment by jameslaneconkling** [comment]
Agreed.  Keep an eye on the TM API for full support for this: https://github.com/hotosm/osm-tasking-manager2/wiki/API

Would require:
- task_details
- lock
- unlock

functionality supported by the API.  Pierre discussed the issues with these (particularly, authenticating 3rd-party apps for post requests--i.e. lock/unlock calls) at the Summit.  Will take some time to develop.
